src/dev.sh

```diff
@@ -361,7 +361,7 @@ bash ./Misc/externals.sh $RUNTIME_ID "Pre-Cache" || checkRC "externals.sh Pre-Ca
 
 if [[ "$CURRENT_PLATFORM" == 'windows' ]]; then
     vswhere=$(find "$DOWNLOAD_DIR" -name vswhere.exe | head -1)
-    vs_location=$("$vswhere" -latest -property installationPath)
+    vs_location=$("$vswhere" -products 'Microsoft.VisualStudio.Product.BuildTools' -latest -property installationPath)
```

Comment:
Not sure if this should be kept or not, but without a product definition this script doesn't work with VS Code and only the build agents installed from Visual Studio.

Reply:
I updated my edited logic here to only try adding the product definition if vs_location is empty, should be no conflict in finding binaries under existing paths